domain-management/src/main/java/org/jboss/as/domain/management/controller/SecureOperationReadHandler.java

```diff
@@ -44,7 +40,6 @@
 public class SecureOperationReadHandler implements OperationStepHandler {
 
     private static final String HIDDEN = "<hidden>";
-    private static final Set<Action.ActionEffect> ADDRESS_EFFECT = EnumSet.of(Action.ActionEffect.ADDRESS);
```

Comment:
+1 The code that used this was factored out to a different class that declared its own constant.